src/components/nodes/pool/pool.vue

```diff
@@ -182,6 +182,7 @@ export default {
     createLaneSet() {
       const laneSet = this.moddle.create('bpmn:LaneSet');
       this.laneSet = laneSet;
+      this.laneSet.set('id', 'laneset_0');
```

Comment:
This ID should be unique, and may be the source of the duplicate ID and unresolved reference errors. We have a module that generates unique IDs (`NodeIdGenerator.js`) that we use for generating unique IDs for all other nodes; you could try using that here as well.

Reply:
```
import NodeIdGenerator from '@/NodeIdGenerator';
const generator = new NodeIdGenerator(this.definitions);
const [laneSetId] = generator.generate();
```